students/kondel_tomasz/lesson_04_unit_testing/distance_test.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import sys
+sys.path.insert(0, '../lesson_03_funtions/')
+from the_length_of_the_segment import distance  # noqa: E402
+
+
+def test_crash_with_TypeError():
+    with pytest.raises(TypeError):
+        (distance(None, 0, 0, 0))
+
+
+def test_crash_with_ValueError():
+    with pytest.raises(ValueError):
+        (distance('aoeu', 0, 0, 0))
+
+
+def test_zero_length():
+    assert(distance(0, 0, 0, 0) == 0)
+
+
+def test_negative_coordinates():
+    assert(distance(-5, -5, -3, -5) > 0)
```

Comment:
Not very strict... `math.fabs(result) < epsilon` would be better. Think about it as something that could save your ass from searching for an error, when you are already using that function. You want to trust your code, so you need to check it hard.